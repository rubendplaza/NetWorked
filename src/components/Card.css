.card {
    border-color: #45A29E;
    /* border-color: #66FCF1; */
    color: #FFFFFF;
    background-color: #1F2833;
    cursor: pointer;
}

