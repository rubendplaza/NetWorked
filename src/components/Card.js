import React from "react";
import "./Card.css";

const Card = ({ name, email, phone, company }) => {
  return (
    <div className="tc dib pa3 ma3 grow shadow-2 card ba bw2">
      <img
        alt="Profile Pic"
        src={`https://avatars.dicebear.com/api/avataaars/${email}.svg?w=200&h=150`}
      />
      <div>
        <h2>{name}</h2>
        <p>{company}</p>
        <p>{phone}</p>
        <p>{email}</p>
      </div>
    </div>
  );
};

export default Card;
