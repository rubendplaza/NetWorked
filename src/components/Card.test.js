import React from "react";
import Card from "./Card";
import { shallow } from "enzyme";

test("expect to render card component", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
