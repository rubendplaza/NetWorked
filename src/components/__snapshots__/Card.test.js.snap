// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div
  className="tc dib pa3 ma3 grow shadow-2 card ba bw2"
>
  <img
    alt="Profile Pic"
    src="https://avatars.dicebear.com/api/avataaars/undefined.svg?w=200&h=150"
  />
  <div>
    <h2 />
    <p />
    <p />
    <p />
  </div>
</div>
`;
